### PR TITLE
initail Neon code

### PR DESCRIPTION
**Pull request by khjj07**
This is a request from Gachon VP-Lab. 
It was reconfigured with Neon referring to the existing SSE code.
This changes can be enabled through -DARM=TRUE in the Cmake build option in the ARM environment.
(Tested on Raspberry Pi 4 environment)

### CODE REVIEW

**Comment by mamthan** [comment]
Thank you for adding the NEON support to the xeve codebase.
We will also run this codebase and add further support.

**Comment by mamthan** [comment]
Hi,

I have integrated the arm neon codebase for our testing on android platform.
I am seeing compliance mismatch, with Arm intriniscs enabled. 
C code has no compliance issue, when I disable ARM intrinsics.

**Comment by mamthan** [comment]
Hi,

I have integrated the arm neon codebase for our testing on android platform.
I am seeing compliance mismatch, with Arm intriniscs enabled. 
C code has no compliance issue, when I disable ARM intrinsics.

File: app/xeve_app.c [hunk 1]
```diff
@@ -38,6 +38,8 @@
 #include <unistd.h>
 #endif
 
+<<<<<<< HEAD
+=======
```
Comment (1480c1) [inline app/xeve_app.c:42]: seems you have merge conflict markers, you may want to clear these up, usually compilers should show an error for these

**Comment by aikiho** [comment]
> Hi,
> 
> I have integrated the arm neon codebase for our testing on android platform. I am seeing compliance mismatch, with Arm intriniscs enabled. C code has no compliance issue, when I disable ARM intrinsics.

We are under debugging on the code pull&requesed. We also found that the code pull&requesed contains mismatch between C and the Neon code. We will shortly debug and update the code and do pull&request again. Thanks for your reviewing.

**Comment by khjj07** [comment]
Hello, I'd like to close this MR at this time. We are working on another piece of code that will support NEON in XEVE. I believe it would be better to close this MR and re-do it later.

**Pull request closed without merging**